lib/partition_setup.pm

```diff
@@ -165,6 +165,9 @@ sub addlv {
     send_key 'ret';    # create logical volume
     assert_screen 'partition-lv-type';
     type_string $args{name};
+
+    send_key($args{thinpool} ? 'alt-t' : $args{thinvolume} ? 'alt-i' : 'alt-o');
```

Comment:
So now in case it's not thin pool or thin volume we additionally press `alt-o`. Can it have any impact on existing tests?

Reply:
It should not http://dhcp128.suse.cz/tests/5120#step/partitioning_lvm_thin_provisioning/47